pkg/products/bamboo/dataset.tf

```diff
@@ -0,0 +1,35 @@
+resource "kubernetes_job" "import_dataset" {
+  metadata {
+    name      = "bamboo-import-dataset"
+    namespace = kubernetes_namespace.bamboo.metadata[0].name
+  }
+  spec {
+    template {
+      metadata {}
+      spec {
+        container {
+          name              = "import-dataset"
+          image             = "alpine:latest"
+          image_pull_policy = "Always"
+          volume_mount {
+            mount_path = "/shared-home"
+            name       = "shared-home"
+          }
+          command = [
+            "/bin/sh", "-c", "apk update && apk add wget && wget ${var.dataset_url} -O /shared-home/${local.dataset_filename}"
+          ]
+        }
+        restart_policy = "Never"
+        volume {
+          name = "shared-home"
+          persistent_volume_claim {
+            claim_name = kubernetes_persistent_volume_claim.atlassian-dc-bamboo-share-home-pvc.metadata[0].name
+          }
+        }
+      }
+    }
+    active_deadline_seconds = "600"
```

Comment:
What is active deadline? Is it different with import timeout? If this is the same is 10 min enough for import the complete given dataset including 20-30% buffer?

Reply:
This just for the job - it is actually too long probably, right now the dataset takes ~10 seconds but because this can be used for any dataset I will keep it 10mins just to be on the safe side.